.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,16 +1,33 @@
+
+---
+name: S2N Issue
+about: Template
+title: ''
+labels: ''
+assignees: ''
```

Comment:
Are these values required?  For assignees the github UI allows us to add with the drop-down options. This would be a duplication in that respect.

Reply:
These allow us to pre-populate the Template, and with GitHub's new template system, we could have one template for bugs that has a pre-populated title `[BUG]` and a feature request template that has `title: '[FEATURE]'`, as an example.

Reply:
Nice add-on! We could probably track a list of keywords like [BUG], [FEATURE] that would trigger our alarms :)